gluon-activations/gluon-activations.md

```diff
@@ -0,0 +1,239 @@
+
+The softsign activation is an alternative to tanh that is also centered at zero but converges asymptotically to -1 and 1 polynomially instead of exponentially. As such, there are a greater range of input values for which the softsign assigns an output of between -1 and 1.
+
+$$ softsign(x) = \dfrac{x}{abs(x) + 1} $$
+
+
+```python
+visualize_activation(data, gluon.nn.Activation('softsign'), "SoftSign")
+```
+
+
+![png](output_11_0.png)
+
+
+The softsign function is not a commonly used activation with most neural networks and suffers even more severely from the vanishing gradient problem as seen in the graph above
+
+## Rectifiers
+
+### ReLU
+ReLU, or Rectified linear unit is the most common activation function in convolutional neural networks and introduces a simple nonlinearity. When the value of the input into ReLU is positive, then it retains the same value. When the value is negative then it becomes zero. In equation form, the ReLU function is given as:
+
+$$ ReLU(x) = \mathtt{max}(0, x) $$
+
+ReLU was introduced to neural networks in the [paper by Hahnloser et al](https://papers.nips.cc/paper/1793-permitted-and-forbidden-sets-in-symmetric-threshold-linear-networks.pdf) and gained widespread popularity after it was shown in the [paper](https://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf) by Alex Krizhevsky et al to perform much better than sigmoid and tanh. This paper also introduced the AlexNet CNN that won the ILSVRC challenge in 2012.
+
+ReLU is the most widely used activation due to it's simplicity and performance across multiple datasets and although there have been efforts to introduce activation functions, many of them described in this tutorial, that improve on ReLU, they have not gained as much widespread adoption
+
+
+
+```python
+visualize_activation(data, gluon.nn.Activation('relu'), "ReLU")
+```
+
+
+![png](output_15_0.png)
+
+
+As shown above, the ReLU activation ameliorates the vanishing gradient problem associated with the sigmoid family of activations, by having a larger (infinite) range of values where it's gradient is non-zero.
+
+### SoftReLU
+
+SoftReLU also known as SmoothReLU or SoftPlus is a nonlinear activation function that takes the form
+
+$$ softReLU(x) = log(1 + e^x)$$
+
+The softReLU can be seen as a smooth version of the ReLU by observing that it's derivative is the sigmoid, seen below, which is a smooth version of the gradient of the ReLU shown above
+
+
+```python
+visualize_activation(data, gluon.nn.Activation('softrelu'), "SoftReLU")
+```
+
+
+![png](output_18_0.png)
+
+
+### Leaky ReLU
+
+Leaky ReLUs are a variant of ReLU that allow a small gradient when the value is negative. The equation for the LeakyReLU is:
+
+$$ LeakyReLU(\alpha, x) = \begin{cases}
+    x,& \text{if } x\geq 0\\
+    \alpha x,              & \text{otherwise}
+\end{cases}$$ 
+
+where $\alpha > 0$ is small positive number. Here is a visualization for the LeakyReLU with $\alpha = 0.01$
```

Comment:
Any chance you could exaggerate alpha for the diagrams, but mention 0.01 is most common (if it actually is).